### DISCUSSION TITLE
How to detect sprite collision

### DISCUSSION THREAD

**Discussion by freezyh** [pixijs/pixijs#8428] [Q&A]
### Environment

- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: _e.g. Chrome 67_
- **OS & Version**: _e.g. Ubuntu 18.04_
- **Running Example**: _url_
How to detect sprite collision，sprites are non-rectangular
![image](https://user-images.githubusercontent.com/26577708/175191665-1f101034-9fda-44f4-bc50-e16862a2d0ca.png)

**Comment by miltoncandelero** [comment]
You should look into SAT: Separating Axis Theorem.
There is a js implementation here: https://github.com/jriecken/sat-js

Pixi doesn't handle collisions for you 😅 
The most you can do is get the bounding box and check if those overlap, but after that you are on your own 😛
